hermes/src/network/rpc/rest.rs

```diff
@@ -24,47 +15,56 @@ use {
         Json,
     },
     axum_extra::extract::Query, // Axum extra Query allows us to parse multi-value query parameters.
+    base64::{
+        engine::general_purpose::STANDARD as base64_standard_engine,
+        Engine as _,
+    },
+    derive_more::{
+        Deref,
+        DerefMut,
+    },
+    pyth_sdk::{
+        PriceFeed,
+        PriceIdentifier,
+    },
 };
 
+#[derive(Debug, Clone, Deref, DerefMut)]
+pub struct PriceIdInput([u8; 32]);
+impl_deserialize_for_hex_string_wrapper!(PriceIdInput, 32);
```

Comment:
May be better to replace with `struct Hex` with const generics.

Reply:
I will address this in a later PR. Added a todo for it.